test.py

```diff
@@ -0,0 +1,27 @@
+from pictureflow.flow import Broadcast, Combine
+from pictureflow.output import DiskOutput
+from pictureflow.transform import Rotate, Scale
+
+import cv2
+import pictureflow as pf
+
+
+inpt = pf.Placeholder(out_type=pf.Image)
```

Comment:
This seems like it's a play around file more than an actual test file do we need it ?

Reply:
@EFF I just created an [issue](https://app.zenhub.com/workspace/o/mentum/pictureflow/issues/8) for this, I'll get on that soon.